tests/Feature/Events/WordpressEventPushTest.php

```diff
@@ -67,15 +68,15 @@ public function date_format_in_events() {
             $mock->shouldReceive('newPost')
                 ->withArgs(function($name, $text, $content) use ($event) {
                     // Check name, and that timestamp doesn't include seconds.
-                    $party_time = substr($event->start, 0, 5) . ' - ' . substr($event->end, 0, 5);
+                    $party_time = substr($event->start_local, 0, 5) . ' - ' . substr($event->end_local, 0, 5) . ' (' . $event->timezone . ')';
```

Comment:
Want to remove the timezone from the string, so this test will need to change.

Reply:
Ok.